### ISSUE TITLE
jest/valid-describe describe.each false positive

### ISSUE THREAD

**Issue by kddeisz** [jest-community/eslint-plugin-jest#463]
I can see #334 and #336 are both related, wasn't sure if you would want it reported on one of those or not. Happy to move it if it would be better elsewhere. Here's the violating code:

```javascript
const cases = [
  ["email", EmailField],
  ["number", NumberField],
  ["password", PasswordField],
  ["string", StringField]
];

describe.each(cases)("%s", (type, FormField) => {
  ...
});
```

It's currently saying `Unexpected argument(s) in describe callback`.
